packages/components/src/VirtualizedList/CellBadge/CellBadge.component.js

```diff
@@ -0,0 +1,20 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import Badge from '../../Badge';
+
+/**
+ * Cell renderer that displays a badge
+ */
+function CellBadge({ cellData, rowIndex }) {
+	return <Badge id={`${rowIndex}`} label={cellData} selected />;
```

Comment:
hard coded-selected : is it for the style ?

Reply:
Yep. I know. It should be retrieved from the columnData. :)